server/service/client.go

```diff
@@ -1978,6 +1987,59 @@ func (c *Client) doGitOpsNoTeamSetupAndSoftware(
 	return softwareInstallers, vppApps, nil
 }
 
+func pluralize(count int, ifSingle string, ifPlural string) string {
+	if count == 1 {
+		return ifSingle
+	}
+	return ifPlural
+}
+
+func (c *Client) doGitOpsLabels(config *spec.GitOps, logFn func(format string, args ...interface{}), dryRun bool) error {
+	persistedLabels, err := c.GetLabels()
+	if err != nil {
+		return err
+	}
+	var numUpdates int
+	var labelsToDelete []string
+	for _, persistedLabel := range persistedLabels {
+		if persistedLabel.LabelType == fleet.LabelTypeBuiltIn {
+			continue
+		}
+		if slices.IndexFunc(config.Labels, func(configLabel *fleet.LabelSpec) bool { return configLabel.Name == persistedLabel.Name }) == -1 {
```

Comment:
Neat!

Reply:
Go has nice package docs, I'll give them that 😉